### PR TITLE
Add core-store.ee_information to default excludes

### LINKED ISSUE #88: Exclude EE Information by default
Hi,

I believe `/config/sync/core-store.ee_information.json` should be excluded by default. Otherwise the license could be commited in plaintext to the repo by accident.

The json file

```json
{
  "key": "ee_information",
  "value": {
    "lastCheckAt": 1677655928783,
    "license": "my-license-in-plaintext"
  },
  "type": null,
  "environment": null,
  "tag": null
}
```

### PR DESCRIPTION
**Pull request by LucaNerlich**
this fixes #88

### CODE REVIEW

**Review by boazpoolman — CHANGES_REQUESTED** [review]
Could you also update the README accordingly?

File: README.md [hunk 1]
```diff
@@ -357,8 +357,9 @@ In the example below you can see how, and also what the default settings are.
 	      excludedTypes: [],
 	      excludedConfig: [
 	        "core-store.plugin_users-permissions_grant",
-          "core-store.plugin_upload_metrics",
-          "core-store.strapi_content_types_schema",
+          	"core-store.plugin_upload_metrics",
+          	"core-store.strapi_content_types_schema",
+		"core-store.ee_information",
```
Comment (LucaNerlich) [inline README.md:362]: image

no idea why github shows the alignment like this

**Comment by LucaNerlich** [comment]
> Could you also update the README accordingly?

of course. Didnt even see that example on it - sorry!

**Review by boazpoolman — CHANGES_REQUESTED** [review]
Could you also add the change on line 429?

File: README.md (hunk 1 above)
Comment (boazpoolman) [reply README.md:362]: LGTM

**Comment by LucaNerlich** [comment]
> Could you also add the change on line 429?

done, sorry for the back and forth. Did not expect these values to be written in the reademe at all 🙈

**Pull request merged**